src/dotnet/ToolManifest/LocalizableStrings.resx

```diff
@@ -129,7 +129,7 @@
 {1}</value>
   </data>
   <data name="ManifestVersion0" xml:space="preserve">
-    <value>Manifest version 0 is not supported. Manifest file path: {0}</value>
+    <value>Manifest version 0 is not supported.</value>
```

Comment:
Why did we decide to remove the extra info?

Reply:
it is redundant. It should be like the following 
```
Invalid manifest file. Path c:\dir\dotnet-tool.json:
  Manifest version 0 is not supported.  **removeing Path c:\dir\dotnet-tool.json**
```